src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -33,7 +33,7 @@ internal static partial class Library
                 BuiltinFunctionsCore.Abs,
                 StandardErrorHandling<NumberValue>(
                     expandArguments: NoArgExpansion,
-                    replaceBlankValues: ReplaceBlankWithZero,
```

Comment:
As far as I know, this is intended for excel compat and the JS impl is wrong. We have a backlog item to bring the JS impl in line with this. 
@CarlosFigueiraMSFT 
![image](https://user-images.githubusercontent.com/69215460/165143562-95f77418-9d60-48c3-923b-129a654974ca.png)

Reply:
That's correct. Moving forward, we will treat blank values as 0 *in numeric context*, which is the case for all math functions. We will introduce a new flag soon in Canvas apps to move to this new "Excel-compat" behavior.

Reply:
Thanks, I'll fix the code and corresponding tests